cli/app.go

```diff
@@ -108,7 +111,14 @@ var app = &cli.App{
 			Aliases: []string{"vvv"},
 			Usage:   "enable debug logging",
 		},
+		&cli.BoolFlag{
+			Name:    quietFlag,
+			Value:   false,
+			Aliases: []string{"q"},
+			Usage:   "suppress warnings; ie version warning, etc",
```

Comment:
I like the idea of a `q` flag, but I don't think what you currently have will silence all warnings. See all usages of `warningf`. Would be cool if we could silence all of those somehow.

Reply:
true. would a more granular `--quiet-update` flag be more explicit for now until we figure out how to silence all warnings?
One option would be to handle this within `warningf` itself, though we would either need to set a global, include the cliContext as a parameter of warningf, or pass the cliContext's quietFlag field in directly.

Reply:
added as ticket: https://viam.atlassian.net/browse/RSDK-6683